app/javascript/emoji_picker/EmojiPicker.vue

```diff
@@ -62,6 +62,12 @@ function listItemClasses(emojiData: EmojiData) {
 function selectEmoji() {
   const symbol = highlightedSearchable.value.symbol;
   navigator.clipboard.writeText(symbol);
-  toast(`Copied ${symbol}`);
+  toast(
+    /*html*/ `
```

Comment:
In conjunction with this [VS Code extension](https://marketplace.visualstudio.com/items?itemName=Tobermory.es6-string-html), this comment provides syntax highlighting for the HTML in this string in VS Code.